ROX-22864: core_bpf does not need support-package (#72821)
openshift/openshift-docs: modules/update-kernel-support-packages.adoc

= Updating kernel support packages in offline mode

Collector monitors the runtime activity for each node in your secured clusters.
To monitor the activities, Collector requires probes in the form of eBPF programs.

With the `CORE_BPF` collection method, the probe is not specific to any kernel version, and can still be used after the underlying kernel

> as been updated. This collection method does not require to provide a support-package or to update it.

Reviewer: ```suggestion
has been updated. This collection method does not require you to provide or update a support package.
```

> On the contraty, when making use of the collection method `EBPF`, the probes are specific to the Linux kernel version installed on the host.

Reviewer: ```suggestion
Instead, when you use the collection method `EBPF`, the probes are specific to the Linux kernel version installed on the host.
```

> The Collector image contains a set of built-in probes for the kernels supported at release time. But later kernels will require newer probes.

Reviewer: ```suggestion
The Collector image contains a set of built-in probes for the kernels supported at release time. However, later kernels will require newer probes.
```
